datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py

```diff
@@ -14,20 +14,19 @@
 
 @click.command("validate-profile", short_help="Validate SNMP profiles", context_settings=CONTEXT_SETTINGS)
 @click.option('-f', '--file', help="Path to a profile file to validate")
-@click.option('-d', '--directory', multiple=True, help="Path to a directory of profiles to validate")
+@click.option('-d', '--directory', 'directories', multiple=True, help="Path to a directory of profiles to validate")
```

Comment:
I would change the option name too and helper for consistency

Reply:
I'm not sure we should since `-d/--directory aDirectory` each option is a single directory